oslabs-beta/KataLog
# Bl/style

##Overview

**Issue Type**
- [] Bug
- []Tech debt

**Description**
Login and Signup pages converted from MUI to Chakra.

**Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix**
Open application (splash page).
Click Login or Signup buttons.
Styling of both pages should match styling of the splash page.

**Previous Behavior**
Login and Signup pages styled using MUI.

**Expected Behavior**
Login and Signup page styling should match the styling present on the splash page.

**How Has This Been Tested**
Opened application (splash page).
Clicked Login and Signup buttons to view styling of Login and Signup pages.

**Screenshots & Videos**
N/A.

**Additional Context**
N/A.
